raiko-host/src/host/host.rs

```diff
@@ -104,8 +104,15 @@ pub fn preflight(
         assert!(blob_hashs.len() >= 1);
         // Currently the protocol enforces the first blob hash to be used
         let blob_hash = blob_hashs[0];
+
+        let l2_chain_spec = get_network_spec(network);
+        let slot_id = block_time_to_block_slot(
+            l1_inclusion_block.header.timestamp.as_limbs()[0],
+            l2_chain_spec.genesis_time,
+            l2_chain_spec.seconds_per_slot,
+        )?;
```

Comment:
Could something like `l1_inclusion_block_no - genesis_block_id` also work to avoid having to use timestamps?

Reply:
Any potential problem using timestamp??
I think using l1_inclusion_block_no is not work as it increases sequentially which is mismatch with the beacon slot id.

Reply:
Just some extra dependencies on block times and time in general that you would not have with block ids